### DISCUSSION TITLE
Steering is reverse when using MT12 surface radio model file from one unit to another

### DISCUSSION THREAD

**Discussion by melwong** [EdgeTX/edgetx#5638] [General]
I experienced the weirdest phenomenon and wonder if it was a bug. I own a Radiomaster MT12 and configured everything but then I just got another one. The first MT12 uses EdgeTX v2.10.0 nightly while the second uses an older version.

I restored the backup from the first to the new unit and the steering becomes reversed. Meaning if I turn left, the wheels go right and vice versa.

So, I went to install EXACTLY the same firmware on my second unit (V2.10.0-nightly) as the first unit. I deleted everything on the SD card and copied over the entire contents from my first MT12 and still the steering is reversed. I tried rebinding but no joy.

This happens when I'm using the same model I used on the first unit. But the steering doesn't go reverse if I create a new model. How come? I checked and my channel output is not inversed.

**Comment by pfeerick** [comment]
Asked on discord, Couple of questions there to be answered ;) https://discord.com/channels/839849772864503828/1298618950073974844

**Reply by melwong** [reply]
Sorry, I just saw them. Surprisingly, I didn't get notifications about the questions. I will monitor it from now.
